src/borg/helpers/misc.py

```diff
@@ -213,3 +213,23 @@ def write(self, s):
                 except OSError:
                     pass
         return len(s)
+
+
+def iter_separated(fd, sep=None, read_size=1024):
```

Comment:
hmm, maybe read_size=4096 (typical fs block size and also less partial processing due to less block boundaries)?

Reply:
Page size makes sense, i wasn't too sure about this number.